src/main/flight/interpolated_setpoint.c

```diff
@@ -62,25 +62,23 @@ FAST_CODE_NOINLINE float interpolatedSpApply(int axis, bool newRcFrame, ffInterp
         const float setpointAcceleration = (setpointSpeed - prevSetpointSpeed[axis]) * pidGetDT();
 
         setpointDeltaImpl[axis] = setpointSpeed * pidGetDT();
-        
+
         const float ffBoostFactor = pidGetFfBoostFactor();
         float clip = 1.0f;
         float boostAmount = 0.0f;
-        if (ffBoostFactor != 0.0f) {
+        if (axis != FD_YAW && ffBoostFactor != 0.0f) {
```

Comment:
Good point. Issues like these are the reason why I think integrated yaw is superior to normal yaw. It just lets us treat all axis the same way.

Reply:
Should this constraint be modified to not remove boost on yaw if integrated yaw is active?

Reply:
To be honest I think yaw might need a separate boost constant due to its different mechanism. So you can leave as is for now.